R/util.R

```diff
@@ -81,11 +81,19 @@ defaults = function(x, defaults) {
 # workaround replacements for other patterns that don't quite do what we need them to
 # (especially when it comes to rvars...)
 
+map_dfr_ = function(data, fun, ...) {
+  # drop-in replacement for purrr::map_dfr
+  do.call(
```

Comment:
I would consider using `bind_rows(lapply(...))` for this as that is safer (handles column differences)